students/itsabhi/Lesson9/oo_mailroom.py

```diff
@@ -0,0 +1,124 @@
+#! /usr/bin/env python3
+
+
+class Donor:
+
+    donor_name = ""
+    donations = []
+
+    def __init__(self, donor, donations):
+        self.donor_name = donor
+        self.donations = donations
+
+    def add_donation(self, amount):
+        self.donations.append(amount)
+
+    def sum_donations(self):
+        return sum(self.donations)
+
+
+class Donors:
+
+    donors = []
+
+    def __init__(self, initial_dlist):
+        self.donors = initial_dlist
+
+    def list_donors(self):
+        # Prints donor name for each Donor object in donors list
+        for d in self.donors:
+            print(d.donor_name)
+
+    def search_donors(self, donor):
+        for d in self.donors:
+            if donor == d.donor_name:
+                return d
+            else:
+                return False
```

Comment:
it's common to return a thing or None for functions like this; returning False makes sense for functions that normally would return a boolean value.